src/picturepark-sdk-v1-angular/src/app/content-picker/content-picker.component.ts

```diff
@@ -54,8 +54,8 @@ export class ContentPickerComponent implements OnInit, OnDestroy {
     this.subscription.add(itemsSubscription);
   }
 
-  public openDetails(itemId: string) {
-    this.dialog.open(DetailsDialogComponent, { data: itemId });
+  public openDetails(item: any) {
```

Comment:
is it posible to know the type of the item?

Reply:
Yes, will fix this in the next round. ContentModel<T> is not exposed.